wasm-runtime/polkadot/src/runtime/governance.rs

```diff
@@ -28,7 +28,8 @@
 use runtime_std::prelude::*;
 use codec::KeyedVec;
 use support::storage;
-use primitives::{AccountID, Hash, BlockNumber, Proposal};
+use primitives::{AccountId, Hash, BlockNumber};
+use primitives::proposal::Proposal;
```

Comment:
can we keep all primitive types in the same module? having to import `Proposal` on a line of its own is unnecessary visual clutter.

Reply:
I disagree; it's better to have the types organized.

Reply:
they're not organised. it's completely arbitrary that `Proposal` should be somewhere other than right in `primitives`.

Reply:
`Function`, `InternalFunction`, and `Proposal` now subtypes of the transaction module.